style/facelift.scss

```diff
@@ -0,0 +1,242 @@
+$font-newspaper: georgia, 'PT Serif', serif;
+$font-ui: 'Avenir Next', 'Montserrat', sans-serif;
+$transition-fast: 0.3s ease;
+$offblack : #333;
+$offwhite : #ededed;
+$accent1  : #F16061;
```

Comment:
i'd be more specific with variable names ultimately. for spec content it's ok, but best to call an orange an orange.